Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8-slim
+WORKDIR /mote
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+RUN sed -i 's|json_cache_location = "/var/cache/httpd/mote/cache.json"|json_cache_location = "./cache.json"|' ./mote/config.py
+EXPOSE 5000
+COPY . .
```

Comment:
How about placing the `COPY` statement before exposing the port, as It'll make the Dockerfile look better. 

What do you think ?

Reply:
+1 to that. I'd do it.